### PR TITLE
Transition to nodejs v14

### PR DESCRIPTION
**Pull request by vpowler**
# Checklist:

- [x] Code follows the style guidelines
- [ ] Documentation updated
- [ ] Corner cases evaluated and handled
- [ ] index.js exports updated
- [ ] Automatic tests added/updated
- [ ] Automatic tests pass
- [x] Sanity check passed
- [x] Self-review passed
- [x] Version updated

### CODE REVIEW

**Review by vitalii427 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "build:sol": "truffle compile all",
     "strip": "babel-node src/tools/strip_contracts.js src/contracts",
     "clean": "rm -fr ./src/contracts/* ./dist/*",
-    "test:units": "mocha",
+    "test:units": "mocha --config=./test/mocharc.json --recursive",
```
Comment (valar999) [inline package.json:20]: Not sure why we need this

File: .travis.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ cache:
   directories:
   - node_modules
 node_js:
-  - '9'
+  - '10'
```
[the remark this answers is not in the document]
Comment (vpowler) [reply .travis.yml:8]: @valar999 Both supported

File: package.json (hunk 1 above)
Comment (vpowler) [reply package.json:20]: this is the result of migration from old mocha. it doesn't work without this.

Comment (vpowler) [reply package.json]: [https://github.com/ambrosus/ambrosus-node-contracts/pull/296/commits/18d2094b418c2516ad52b1c8a12831206558dac2](url)

**Review by seagiv — APPROVED** [review]
